eZ/Bundle/EzPublishCoreBundle/Features/QueryController/query_controller.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Query controller
+    In order to simplify listing items from the repository
+    As a developer
+    I want to run repository queries from content views
+
+Scenario: A content view can be configured to run and render a query
+    Given the following configuration:
+    """
+    ezpublish:
+        default:
+            location_view:
+                full:
+                    folder:
+                        match:
+                            Identifier\ContentType: 'folder'
+                        controller: ez_query:locationQueryAction
+                        template: '@EzPlatformBehatBundle/Resources/views/query_controller.html.twig'
+                        params:
+                            query: 'LocationChildren'
+                            queryParameters:
+                                parentLocationId: '@=location.id'
+                            variable: 'children'
+    """
+    And the following QueryType definition in "src/AppBundle/QueryType/LocaitonChildrenQueryType.php":
+    """
+    <?php
+    namespace AppBundle\QueryType;
+
+    use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+    use eZ\Publish\API\Repository\Values\Content\Query\Criterion\ParentLocationId;
+    use eZ\Publish\Core\QueryType\QueryType;
+
+    class LocationChildrenQueryType implements QueryType
+    {
+        public function getQuery(array $parameters = [])
+        {
+            return new LocationQuery([
+                'filter' => new ParentLocationId($parameters['parentLocationId']),
+            ]);
+        }
+
+        public function getSupportedParameters()
+        {
+            return ['parentLocationId'];
+        }
+
+        public static function getName()
+        {
+            return 'LocationChildren';
+        }
+    """
+    When I view a folder
+    Then the LocationChildren Query is built using the folder's location id as the parentLocationId parameter
+     And a Location Query is ran with the LocationChildren query
+     And the Query results are set to the "children" twig variable
```

Comment:
This doesn't actually say that the children of the given location id are being returned as results. Do you consider that irrelevant to the feature, since it's up to the custom QueryType?